### ISSUE TITLE
No icon in debian based distros

### ISSUE THREAD

**Issue by pistolparody** [nukeop/nuclear#1660]
**Platform:**
```
..............                                     pickle@bizness 
            ..,;:ccc,.                             -------------- 
          ......''';lxO.                           OS: Kali GNU/Linux Rolling x86_64 
.....''''..........,:ld;                           Kernel: 6.6.15-amd64 
           .';;;:::;,,.x,                          Uptime: 23 hours, 6 mins 
      ..'''.            0Xxoc:,.  ...              Packages: 3132 (dpkg) 
  ....                ,ONkc;,;cokOdc',.            Shell: zsh 5.9 
 .                   OMo           ':ddo.          Resolution: 1440x900 
                    dMc               :OO;         DE: Plasma 5.27.10 
                    0M.                 .:o.       WM: kwin 
                    ;Wd                            Theme: [Plasma], Breeze [GTK2/3] 
                     ;XO,                          Icons: [Plasma], Flat-Remix-Blue-Dark [GTK2/3] 
                       ,d0Odlc;,..                 Terminal: konsole 
                           ..',;:cdOOd::,.         CPU: 11th Gen Intel i7-11700 (16) @ 4.800GHz 
                                    .:d;.':;.      GPU: Intel RocketLake-S GT1 [UHD Graphics 750] 
                                       'd,  .'     Memory: 4221MiB / 31866MiB 
                                         ;l   ..
                                          .o                               
                                            c                              
                                            .'
                                             .


```
**[v0.6.33](https://github.com/nukeop/nuclear/releases/tag/v0.6.33)**

**Description of the issue:**
while app is not opened, it has no icon by default. i can pin it to bottom bar, but the icon shows only if the app is opened.
it means that somehow the authorized application which is nuclear, is not mapped correctly to the app icon.

**How I fixed it**
I could fix this issue by manually mapping the application to where I downloaded the app icon.
note that i could not find the app icon in my os after installation and successful run.
so i had to manually download it from https://github.com/nukeop/nuclear/tree/master/packages/app/resources/media

**Proposed solution**
after installation, add the icon to /usr/share/icons/nuclear, and then map the application correctly to the icon.
I don't know how to do it, but i think it just might work.

further information can be provided if needed.

**Comment by nukeop** [comment]
Thank you for the detailed description. Did you install it from the .deb package?

**Comment by nukeop** [comment]
Hopefully solved by #1661

Please reopen if it still occurs.
